tensorboard/BUILD

```diff
@@ -317,6 +317,7 @@ tf_web_library(
         "//tensorboard/webapp/widgets/line_chart_v2/lib/worker:chart_worker.js",
     ],
     path = "/",
+    suppress = ["*"],
```

Comment:
Webfiles checker is incapable of understanding the URL and matches filename against a path with query parameter. Because we are not using Closure Compiler, this is effectively only suppressing the Webfiles checker which I think is good enough.

Reply:
Hmm, okay. It seemed strange to me because I would have guessed that if
the webfiles validator can’t resolve query strings, whatever actually
does the webfile merging will also fail. And if we’re not doing webfile
merging, then I don’t see why the webfiles validator is even being run.
But I’ll defer to you if you think that this is okay.

Might we perhaps at least change to `suppress = ["strictDependencies"]`?